src/windows/views/blender_listview.py

```diff
@@ -174,7 +174,10 @@ def currentChanged(self, selected, deselected):
             elif param["type"] == "color":
                 # add value to dictionary
                 color = QColor(param["default"])
-                self.params[param["name"]] = [color.redF(), color.greenF(), color.blueF()]
+                if param.get("name") == "diffuse_color":
+                    self.params[param["name"]] = [color.redF(), color.greenF(), color.blueF(), color.alphaF()]
+                else:
+                    self.params[param["name"]] = [color.redF(), color.greenF(), color.blueF()]
```

Comment:
Would it be easier to just set _all_ colors in RGBA format, rather than special-casing? I mean, the [QColor docs say](https://doc.qt.io/qt-5/qcolor.html#alpha-blended-drawing) that fully opaque is the default, so `alphaF()` should just be `1.0` for any solid, non-transparent colors. Couldn't hurt to just always pass it in, and it might become useful later if we decide to add alpha to any other color params.

Reply:
This was mainly for the preview image... so it has more contrast (instead of the background color of the widget... or whatever). But yes, we could probably always default to RGBA and then adjust the widget color in Qt to handle that (I think). It would need some experimenting.

Reply:
I tried this, turns out it doesn't work. Blender is very picky about what color parameters include an alpha value and which don't. Get it wrong and it's either:
```
ValueError: bpy_struct: item.attr = val: sequences of dimension 0 should contain 4 items, not 3
```
or
```
ValueError: bpy_struct: item.attr = val: sequences of dimension 0 should contain 3 items, not 4
```